### ISSUE TITLE
Srcset support ?

### ISSUE THREAD

**Issue by SheepDomination** [nicolas-t/Chocolat#20]
Does Chocolat support srcset ?

**Comment by nicolas-t** [comment]
not yet.
but that's a good idea :+1:

**Comment by SheepDomination** [comment]
I'm desperately seeking a lightbox with this capability, immediately.  You couldn't add it very soon ?

**Comment by nicolas-t** [comment]
You want to chose the image depending of the screen width ?
I'll give it a look, maybe it can be easily implemented.

Best regards

**Comment by SheepDomination** [comment]
Yes, for example here is a snippet of my HTML code;

 <a href="/wdp/wip/paperpile.png" class="firstimg">
                   <img src="/wdp/wip/images/paperpile.png"
                    srcset="/wdp/wip/images/brick.jpg 320w,
                    /wdp/wip/images/bulding.jpg 560w,
                    /wdp/wip/images/rock.jpg 900w"
                      sizes="(min-width: 320px)"
                     alt="" title="">

The anchor is first, but strictly after the img src, is a srcset, therefore; when the page is at a 320w the brick image should be shown, when the page is at a 560w, the building image is shown. Although the mark-up is not nested within a <picture> tag.

**Comment by JesseZ1983** [comment]
I'm trying to use this with srcset as well, but can't get it to work.

I've tried to NOT set the <a href> in the html....rather only set it with js based on whatever the img's src ends up being. Here's the simple code:

		$('.lightbox').each(function() {
			var x = $(this).find('img').attr('src');
			$(this).find('a').attr('href', x.toString()); 
		})

The resulting <a href> tag is correct (same as if I manually set the html), but it doesn't load. Very strange. Can't seem to figure out why this would be.
